backend/src/main/java/com/jobwait/stat/StatRefreshOfficer.java

```diff
@@ -0,0 +1,33 @@
+package com.jobwait.stat;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import com.jobwait.domain.Stat;
+
+public class StatRefreshOfficer {
+    // constants
+    private static Duration statLifetime = Duration.ofMinutes(60);
+
+    // stat id to timestamp
+    private static Map<String, Instant> generateIdStampMap() {
+        return Stats.known.stream().map(s -> s.id)
+                .collect(Collectors.toMap(Function.identity(), _ -> Instant.now(), (_, _) -> {
```

Comment:
do you prefer `Function.identity()` over `s -> s` ? just a question of preference I suppose

Reply:
Funny that the "more readable" version is like triple the character count. I have no strong opinion on this, I'd be fine either way

Reply:
just leave it then honestly, no biggie